bring/pom.xml

```diff
@@ -30,5 +32,10 @@
             <artifactId>slf4j-api</artifactId>
             <version>${slf4j-api.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.2.11</version>
```

Comment:
please extract the version to properties

Reply:
Done